### ISSUE TITLE
Changing the browser used when opening links externally

### ISSUE THREAD

**Issue by kyptin** [MiT-Ufa/Slacker#5]
Hey, thanks for writing this! Neat and useful program. :-)

Is these a way to change which browser links are opened in? I currently use Chrome, but links open in Firefox. I found the settings file at `~/.config/MiT-Ufa/Teamwords.conf`, but there doesn't appear to be any such option. I also looked at the source, and didn't see any such option being used.

What do you think?

**Comment by Snegovikufa** [comment]
@kyptin links should open in system's default browser. What OS do you use?

**Comment by kyptin** [comment]
Ah, that's an important detail that I forgot to mention! I use FreeBSD.

I don't think FreeBSD itself has a default browser. I figured out that I can set the `BROWSER` environment variable, and if that's set in the TeamWords process, TeamWords honors it. So that solves my problem. I'm interested to know how TeamWords/QT/whatever chooses a browser if there's no environment variable set, but I suppose that's not necessary for this ticket. Closing for now. And thanks for your help!
